sqldelight-gradle-plugin/src/test/fixtures/well-formed-selects/expected/com/sample/HockeyPlayerModel.java

```diff
@@ -291,6 +291,66 @@ public T map(@NonNull Cursor cursor) {
     }
   }
 
+  interface Join_friendsModel {
```

Comment:
Future improvement would be to detect if the union is on the same table (or view) and instead of exploding the table into its result columns, use the table.

This interface is a duplicate of the hockey player table.